plugins/resizimg/resize.with.canvas.js

```diff
@@ -0,0 +1,221 @@
+"use strict";
```

Comment:
Should this file wrapped into a self-invoking function to avoid some contamination?